frontend/web/components/navigation/Nav.tsx

```diff
@@ -0,0 +1,389 @@
+            <Flex className='flex-row px-2 '>
+              {AccountStore.getUser() ? (
+                <React.Fragment>
+                  <nav className='mt-2 mb-1 space flex-row hidden-xs-down'>
+                    <Row className='gap-2'>
+                      <Link data-test='home-link' to={'/organisations'}>
+                        <img
+                          style={{
+                            height: 24,
+                            width: 24,
+                          }}
+                          src='/static/images/nav-logo.png'
+                        />
+                      </Link>
+                      {!(isOrganisationSelect || isCreateOrganisation) && (
+                        <div className='d-flex gap-1 ml-1 align-items-center'>
+                          <div
+                            className={
+                              !!activeProject && !!projectId
+                                ? 'd-none d-md-block'
+                                : ''
+                            }
+                          >
+                            <BreadcrumbSeparator
+                              projectId={projectId}
+                              hideSlash={!activeProject}
+                              focus='organisation'
+                            >
+                              <NavLink
+                                id='organisation-link'
+                                data-test='organisation-link'
+                                activeClassName='active'
+                                className={classNames('breadcrumb-link', {
+                                  active: !projectId,
+                                })}
+                                to={Utils.getOrganisationHomePage()}
+                              >
+                                <div>
+                                  {AccountStore.getOrganisation()?.name}
+                                </div>
+                              </NavLink>
+                            </BreadcrumbSeparator>
+                          </div>
+                          {!!activeProject && !!projectId && (
+                            <BreadcrumbSeparator
+                              projectId={projectId}
+                              hideSlash
+                              focus='project'
+                            >
+                              <NavLink
+                                to={`/project/${activeProject.id}`}
+                                id='project-link'
+                                activeClassName='active'
+                                className={'breadcrumb-link active'}
+                              >
+                                <div>{activeProject.name}</div>
+                              </NavLink>
+                            </BreadcrumbSeparator>
+                          )}
+                        </div>
+                      )}
+                    </Row>
```

Comment:
Sorry to be picky, I also think the whole Navigation/Header part would gain in visibility by extracting these in different parts. E.g here `SelectOrgAndProject` and the below part `HeaderTopBarActions` so we have a human readable way of getting what the conditions render.

I think we can achieve something like this:
```
{!(isOrganisationSelect || isCreateOrganisation) &&
<SelectOrgAndProject />
}
<HeaderTopBarActions />
{showNav && (
   <OverFlowNav>
   {isActiveProject ? 
   <ProjectNavBar/> : <OrganisationNavBar/>)}
   </OverFlowNav>
)}

   ... rest
```